scripts/update-snapshots.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+LOCAL_REST_API_BINDING_PORT=15051
+SNAPSHOT_DOWNLOAD_SCRIPT="./scripts/fetch-deposit-snapshot.sh"
+
+OUT=${1:?Must specify destination directory for snapshots}
+
+echo $'\nChecking that required environment variables are set:'
+all_env_set=true
+require_env() {
+  echo -n "Checking environment variable: $1... "
+  env_name="$1"
+  if [ -z ${!1+set} ]; then
+    echo "FAILED"
+    all_env_set=false
+  else
+    echo "OK"
+  fi
+}
+
+require_env "SERVER_GOERLI_URL"
+require_env "SERVER_MAINNET_URL"
+require_env "SERVER_SEPOLIA_URL"
+require_env "SERVER_GNOSIS_URL"
+require_env "MAINNET_SSH_KEY"
+
+if [[ "$all_env_set" = false ]]; then
+  echo "Not all required environment variables are set. Following environment variables should be set:"
+  echo "SERVER_GOERLI_URL: ssh url to Goerli server with Teku like 'user@8.8.8.8'"
+  echo "SERVER_MAINNET_URL: ssh url to Mainnet server with Teku like 'user@8.8.8.8'"
+  echo "SERVER_SEPOLIA_URL: ssh url to Sepolia server with Teku like 'user@8.8.8.8'"
+  echo "SERVER_GNOSIS_URL: ssh url to Gnosis server with Teku like 'user@8.8.8.8'"
+  echo "MAINNET_SSH_KEY: path to SSH key required for Mainnet server like '~/.ssh/secret-key.pem'"
```

Comment:
I'm not really sure why mainnet is special here. We'd likely get these snapshots from our mainnet-archive box so it has the same security setup as the testnet boxes.

Reply:
It makes things even simpler and more secure. Checked it with archive node, all good, updated